packages/ingest/test/test-granule.js

```diff
@@ -553,3 +608,240 @@ test('handleDuplicateFile calls S3.moveObject with expected arguments if duplica
     }
   );
 });
+
+test('moveGranuleFile moves a granule file and updates postgres', async (t) => {
+  // Create granule in postgres
+  const bucket = t.context.internalBucket;
+  const secondBucket = t.context.destBucket;
+  const testPrefix = cryptoRandomString({ length: 10 });
+  const fileName = cryptoRandomString({ length: 10 });
+  const key = `${testPrefix}/${fileName}`;
+
+  const granulePgModel = new GranulePgModel();
+  const filePgModel = new FilePgModel();
+  const granuleId = cryptoRandomString({ length: 6 });
+
+  // eslint-disable-next-line camelcase
+  const [cumulus_id] = await granulePgModel.create(
```

Comment:
```suggestion
  const [granuleCumulusId] = await granulePgModel.create(
```

You don't have to name the de-structured variable `cumulus_id`. And then you can avoid annoying ESlint warnings

Reply:
This sort of thing was more about using it for an implicit  parameter later (e.g. L 663), but that's a style thing and probably not a good choice on my part, *particularly* given that granule_cumulus_id was the more used key. 

d182fc63d